ios/FluentUI/Separator/Separator.swift

```diff
@@ -45,6 +45,14 @@ open class Separator: UIView, TokenizedControlInternal {
         return fluentTheme.color(.stroke2)
     }
 
+    open override func willMove(toWindow newWindow: UIWindow?) {
+        super.willMove(toWindow: newWindow)
+        guard let newWindow else {
+            return
+        }
+        tokenSet.update(newWindow.fluentTheme)
```

Comment:
remindme: we don't need to set backgroundcolor = tokenset[.color] ?

Reply:
Nope since the background color is set in the `registerOnUpdate` callback it will run when the token set is updated.